.github/workflows/ci-workflow.yml

```diff
@@ -43,6 +43,11 @@ env:
     -pl -minifi/minifi-assembly
     -pl -nifi-system-tests
 
+
+concurrency:
+  group: "${{ github.head_ref }}"
```

Comment:
Is there a reason for using `github.head_ref` as opposed to `github.ref`?  The documentation indicates that `github.head_ref` is only available on `pull_request`, but this workflow can be triggered from either a `pull_request` or `push`.

Reply:
Didn't think things through all the way.  Thanks, I will make that change.